lisa/src/main/java/it/unive/lisa/analysis/impl/numeric/Sign.java

```diff
@@ -261,4 +264,20 @@ protected Satisfiability satisfiesTernaryExpression(TernaryOperator operator, Si
 			ProgramPoint pp) {
 		return Satisfiability.UNKNOWN;
 	}
+
+	@Override
+	protected ValueEnvironment<Sign> assumeBinaryExpression(
+			ValueEnvironment<Sign> environment, BinaryOperator operator, ValueExpression left,
+			ValueExpression right, ProgramPoint pp) throws SemanticException {
+		switch (operator) {
+		case COMPARISON_EQ:
```

Comment:
Can't we assume also other comparison operations? For instance, if we have `+ > -` we can infer the bottom value. This is also true for the other non relational domains. 

I'm also concerned about the fact that `left` and `right` are still expressions, isn't it better to already have them as instances of `Sign`?